lms/envs/common.py

```diff
@@ -2298,3 +2301,15 @@
 
 # This is to check the domain in case of preview.
 PREVIEW_DOMAIN = 'preview'
+
+# Sets the maximum number of courses listed on the homepage
+# If set to None, all courses will be listed on the homepage
+HOMEPAGE_COURSE_MAX = 9
+
+
+if FEATURES.get('ENABLE_COURSE_DISCOVERY'):
```

Comment:
@marjev - We're going to need to rethink this - common.py is the first thing to get processed - so we can't set `ENABLE_COURSE_DISCOVERY` to `True` here without adding `False` elsewhere. (So, I set it to `False` while rebasing).

However, conditional code will never run because we're in the same file - so it will only go one way or the other and so this pathway is not usable.

I'd recommend setting `HOMEPAGE_COURSE_MAX = None` in common.py and then set the other overrides for devstack.py